modules/prebidServerBidAdapter/index.js

```diff
@@ -984,6 +984,12 @@ Object.assign(ORTB2.prototype, {
           });
           bidObject.requestTimestamp = this.requestTimestamp;
           bidObject.cpm = cpm;
+          bidObject.bidderCode = seatbid.seat;
+          if (bidRequest?.bidder) {
```

Comment:
The intention is that bidObject.adapterCode is supposed to be defined by bid.ext.prebid.meta.adaptercode as a first priority, then default to seatbid.seat.

Reply:
Hi @bretg ,
What you are pointing out is bidderCode, which should be similar to seatbid.seat. 
I have made changes to consume adapterCode from bid.ext.prebid.meta.adaptercode as first priority, then bidRequest.bidder and then seatbid.seat

Reply:
this is better @pm-azhar-mulla , but I think this deserves a unit test or 6.